src/renderer/components/configure-recording-set-page/set-recording-list-configuration.tsx

```diff
@@ -32,7 +43,10 @@ const SetRecordingListConfiguration: FC<SetRecordingListConfigurationProps> = ({
             className={'w-100'}
             style={{ paddingRight: '0.75rem' }}
             value={chosenBuiltInList}
-            onChange={(e): void => setChosenBuiltInList(e.target.value)}>
+            testId={'test'}
```

Comment:
Is `testId` necessary? It appears to me that's the `escape hatch` provided by `react-test-library`. Can you do without it?

Reply:
Also, shouldn't the ID be more unique?

Reply:
I changed the ID to make it more unique, but unfortunately it is necessary to make the dropdown selectable in the tests I'm running.  I need to select it/feed input to it in order to run the tests, and because it doesn't have default text (and there's another selector of the same type on the screen) i need the ID to make it identifyable